docs/news/news-v5.0.x.rst

```diff
@@ -4,14 +4,21 @@ Open MPI v5.0.x series
 This file contains all the NEWS updates for the Open MPI v5.0.x
 series, in reverse chronological order.
 
-Open MPI version v5.0.2rc1
+Open MPI version v5.0.2rc2
 --------------------------
-:Date: 24 January 2024
+:Date: 31 January 2024
 
 - Internal PMIx and PRRTe versions:
 
-  - PMIx (v4.2.8-3-ge68bdfcf). Commit hash: ``e68bdfcf4133cb4c1af31c591bb1d26ed63fe911``.
-  - PRRTE (v3.0.3-5-g8bd181af08). Commit hash: ``8bd181af0836375e66c741c1c94364d8a44918e5``.
+  - PMIx (v4.2.8). Commit hash: ``d1c1ed0c2e64f19ad49291241a43630ea7fdce28``.
+  - PRRTE (v3.0.3). Commit hash: ``e383f5ad70c2633420d3425e9fb67e69b6bfd9c4``.
+
+  .. note:: Due to the urgency of 5.0.2 release, the following modules are
+            pinned to the same versions as 5.0.1: PMIx(v4.2.8), PRRTe(v3.0.3).
+            This ensures the modules always refer to stable releases that can
+            be matched by external installations.
+            As a result, changes in newer PMIx/PRRTe are excluded, notably
+            PMIX_QUERY_PSET_MEMBERSHIP query bugfixes.
```

Comment:
```suggestion
  .. note:: Due to the timing of Open MPI v5.0.2 release, the PMIx and PRRTE versions that are internally bundled in the Open MPI distribution are the same versions as in the Open MPI v5.0.1 release: PMIx v4.2.8 and PRRTE v3.0.3.
            As a result, changes and bug fixes in newer PMIx and PRRTE reases not included in Open MPI v5.0.2, notably:
           * PMIX_QUERY_PSET_MEMBERSHIP queries can fail
           * ...list other known issues...
```

We should list all the known issues.  This doc is Google-able, after all.

Reply:
Updated